src/parser/jobs/whm/modules/Lilies.tsx

```diff
@@ -0,0 +1,159 @@
+import {Plural, Trans} from '@lingui/react'
+import Color from 'color'
+import {DataLink} from 'components/ui/DbLink'
+import {ActionKey} from 'data/ACTIONS'
+import {Event} from 'event'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {CounterGauge, Gauge as CoreGauge, TimerGauge} from 'parser/core/modules/Gauge'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const LILY_MAX_STACKS = 3
+const LILY_INTERVAL_600 = 30000
+const LILY_INTERVAL_610 = 20000
+
+const MISERY_COST = 3
+
+const GRAPH_DISPLAY_SETTINGS = {
```

Comment:
Not much value to bundling these, I'd merge them so you define them outright as they're needed with the fade inline, ie `FOO: Color(#696969).fade(0.5)` or whatever, since most of these are single use. That way you can also have a distinct colour for the timer if it's better for UX too.